backend/src/main/java/com/github/liliancharlotte/controller/UserController.java

```diff
@@ -0,0 +1,32 @@
+package com.github.liliancharlotte.controller;
+
+import com.github.liliancharlotte.model.User;
+import com.github.liliancharlotte.model.UserRequest;
+import com.github.liliancharlotte.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
Da dieser Controller nur für `/api/user/...`-Requests zuständig sein soll, kannst Du hier oben direkt `user` mit dazu schreiben und es Dir dafür unten pro Methode sparen.
```suggestion
@RequestMapping("/api/user")
```